lib/cluster/deployment.rb

```diff
@@ -63,7 +63,7 @@ def self.execute_chef_recipes_on_layers(recipes: [], layers: [], custom_json: ''
     end
 
     def self.update_dependencies
-      run_command_on_layers(command: 'dependencies')
+      run_command_on_layers(command: 'update_dependencies')
```

Comment:
All this does is run a default chef 11 recipe provided by opsworks. chef 11.10 is not going away soon in opsworks AFAIK and an upgrade to chef 12 is a big undertaking.  

The reason it's being discontinued is because there are basically no default chef recipes in a chef 12 stack, as chef 12 introduces a bunch of changes that make it easier to use completely native chef.

So in chef 12 stacks, you'll use whatever cookbooks and recipes you want - including one to apply updates. 

I see no reason NOT to use the builtin updater for now.